recipes/larray/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "larray" %}
+{% set version = "0.20" %}
+{% set sha256 = "d7f825f29de27d788c2a95c8950dc9638fa38836294eada98fc7ee28eb9534a8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/liam2/larray/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >=1.10
+    - pandas
+
+  run:
+    - python
+    - numpy >=1.10
+    - pandas
+
+test:
+  imports:
+    - larray
+
+about:
+  home: http://github.com/liam2/larray
+  license: GPL
+  license_file: COPYING
+  summary: "Labeled N-dimensional array."
+
+extra:
+  recipe-maintainers:
+    - alixdamman
+    - gdementen
```

Comment:
@gdementen are you ok being added as a con-maintainer of the larray package on conda-forge?